### ISSUE TITLE
Auto Object Spawner script

### ISSUE THREAD

**Issue by kawasaks** [UnityTechnologies/PlaygroundProject#36]
i am trying to teach my son how to make his first code by following this tutorial (https://unity3d.com/learn/tutorials/topics/2d-game-creation/2d-games-non-programmers?playlist=17093) but seems like update version is missing "Auto Object Spawner" script or either replaced with something else that i can't find in the update version. we are so close to complete the tutorial and don't want to put him down with all the work we have done so far. it will be kind if anyone can help to guide which script is replaced or where i can find it.

thank you for your help in advance.

**Comment by Abhi123378** [comment]
Yes Auto Object Spawner is not available in the updated GitHub repo. But instead you can use the ObjectCreatorArea script. It provides the same functionality as the Auto Object Spawner.
